.github/gh-actions-self-hosted-runners/helper-functions/cloud-functions/monitorRunnersStatus/index.js

```diff
@@ -70,6 +88,7 @@ async function monitorRunnerStatus() {
 }
 
 functions.http('monitorRunnerStatus', (req, res) => {
+    validateEnvSet(["APP_ID","PEM_KEY","CLIENT_ID","CLIENT_SECRET","APP_INSTALLATION_ID","ORG"])
```

Comment:
I'd suggest to set the ENV_VARS as a constant at the top of the script. Same case for the rest of the files. THank you!

Reply:
Oki, will implement the changes, thanks!